docs/reference/quick-start.asciidoc

```diff
@@ -0,0 +1,284 @@
+
+. Extract the downloaded file.
++
+Once you've extracted the file, {es} is ready to run.
+
+. In your terminal, navigate to the newly extracted elasticsearch-{version}
+folder.
+
+. From the elasticsearch-{version} folder, run `bin/elasticsearch`. If using
+Windows, run `bin\elasticsearch.bat` instead.
+
+If all went well, you should see {es} logs in your terminal window.
+Congratulations, you've started {es}!
+
+To stop {es}, you can press `Ctrl + C` in this terminal window. You'll want
+to open a new terminal window for the rest of these steps.
+
+[float]
+=== How to talk to {es}
+While you can interact with {es} in many ways, the examples on this page
+use {es}'s REST API. The REST API offers a few advantages:
+
+* You can talk to {es} using `curl` commands in your terminal.
+* You can use standard HTTP request methods, like `GET`, `POST`, `PUT`, and `DELETE`.
+* Responses from the REST API provide standard HTTP status codes, like `200 OK`
+or `400 Bad Request`. These codes can be helpful when diagnosing a problem or
+figuring out if a request succeeded.
+
+[float]
+=== Index a document
+
+{es} stores data as JSON objects called documents. When adding a new document,
+you must assign it to a collection called an _index_. Usually an index contains
+documents of a similar type, like customer profiles or products in a catalog.
+
+The index not only stores your documents but makes them searchable. Indices are
+so important that adding new documents to {es} is called _indexing_.
+
+To keep your data organized, each document in an index requires a unique ID. You
+can provide this ID when you index a document or let {es} generate one for
+you.
+[float]
+==== Provide a document ID
+
+To index a document with a specific ID, use the PUT method with the
+<<docs-index_, index API>>.
+
+This example command adds a document with an ID of `1` to the `customer` index.
+This command will also create the `customer` index if it doesn't already exist.
+
+[source, js]
+--------------------------------------------------
+PUT /customer/_doc/1
+{
+    "firstname": "Amber",
+    "lastname": "Duke",
+    "age": 32,
+    "employer": "Pyrami"
+}
+--------------------------------------------------
+// CONSOLE
+// TESTSETUP
+
+You can confirm {es} used your ID by checking the `\_id` field of the response.
+Here's an example response, formatted to be more readable.
+
+[source, js]
+--------------------------------------------------
+{
+  "_index" : "customer",
+  "_type" : "_doc",
+  "_id" : "1",
+  "_version" : 2,
+  "result" : "updated",
+  "_shards" : {
+    "total" : 2,
+    "successful" : 1,
+    "failed" : 0
+  },
+  "_seq_no" : 1,
+  "_primary_term" : 1
+}
+--------------------------------------------------
+// TESTRESPONSE
+
+[float]
+==== Autogenerate a document ID
+
+To add a document without providing an ID, use the POST method with the
+<<docs-index_, index API>>. {es} will automatically generate an ID for the
```

Comment:
I generally try to avoid future tense, but it's not a hard rule.

```suggestion
<<docs-index_, index API>>. {es} automatically generates an ID for the
```